index.html

```diff
@@ -217,12 +216,13 @@ <h3>Research</h3>
       </div>
     </div>
 
-    <div class="registration" id="registration">
+    <div class="registration">
       <div class="container">
         <!-- Header -->
-        <div class="registration-header" style="margin-bottom: 2%;">
+        <div class="registration-header" id="registration"> style="margin-bottom: 2%;">
```

Comment:
i think u accidentally added the bracket here after "registration"

Reply:
Kill 🤦‍♀️